sdk/communication/Azure.Communication.Common/CHANGELOG.md

```diff
@@ -1,17 +1,18 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-02-09)
 
 ### Added
 - Added `MicrosoftTeamsUserIdentifier`.
 
 ### Breaking Changes
 - Renamed `CommunicationUserCredential` to `CommunicationTokenCredential`.
-- Replace `CommunicationTokenCredential(bool refreshProactively, Func<CancellationToken, string> tokenRefresher,Func<CancellationToken, ValueTask<string>>? asyncTokenRefresher = null, string? initialToken = null)` 
+- Replaced `CommunicationTokenCredential(bool refreshProactively, Func<CancellationToken, string> tokenRefresher,Func<CancellationToken, ValueTask<string>>? asyncTokenRefresher = null, string? initialToken = null)`.
```

Comment:
Now we have both "Replace...with" syntax and "Renamed...to" syntax. Pick one but not both. ;-)

Reply:
I've stuck to `Rename` when it's a pure rename, and `replaced` when the signature changed functionally as it did here.